pkg/virt-handler/hotplug-disk/mount.go

```diff
@@ -346,6 +349,22 @@ func (m *volumeMounter) isDirectoryMounted(vmiStatus *v1.VirtualMachineInstanceS
 // isBlockVolume checks if the volumeDevices directory exists in the pod path, we assume there is a single volume associated with
 // each pod, we use this knowledge to determine if we have a block volume or not.
 func (m *volumeMounter) isBlockVolume(vmiStatus *v1.VirtualMachineInstanceStatus, volumeName string) bool {
+	// First evaluate the migrated volumed. In the case of a migrated volume, virt-handler needs to understand if it needs to consider the i
+	// volume mode of the source or the destination. Therefore, it evaualtes if its is running on the destiation host or otherwise, it is the source.
```

Comment:
```suggestion
	// volume mode of the source or the destination. Therefore, it evaluates if its is running on the destination host or otherwise, it is the source.
```